README.md

```diff
@@ -6,3 +6,30 @@
 
 ## Overview
 A flexible and easy way to register, manage and group your applications.
+
+## Prerequisities
+
+- [Docker](https://www.docker.com/get-started)
+- [Minikube](https://github.com/kubernetes/minikube) 1.0.1
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.12.5
+- [Helm](https://github.com/kubernetes/helm) 2.10.0
+- [Kyma CLI](https://github.com/kyma-project/cli) master
+
+## Installation
+
+To install Compass locally, run:
+
+```bash
+./installation/scripts/run.sh
+```
+
+## Tests
+
+To run tests, install the Compass and run:
+```bash
+./installation/scripts/testing.sh
```

Comment:
Do we need to keep the structure as it is in Kyma? We don't have installer, so there is no requirement to do so. 

In my opinion we should have two simple scripts on root of the repository:

`install-minikube.sh`
`test.sh`

I think we should discuss it in our team.